kuma/search/filters.py

```diff
@@ -162,50 +161,36 @@ def filter_queryset(self, request, queryset, view):
             filter_tags = serialized_filter['tags']
             filter_operator = Filter.OPERATORS[serialized_filter['operator']]
             if serialized_filter['slug'] in view.selected_filters:
-
                 if len(filter_tags) > 1:
                     tag_filters = []
                     for filter_tag in filter_tags:
                         tag_filters.append(F('term', tags=filter_tag))
-                    active_filters.append(reduce(filter_operator, tag_filters))
+                    active_filters.append(F(filter_operator, tag_filters))
                 else:
                     active_filters.append(F('term', tags=filter_tags[0]))
 
             if len(filter_tags) > 1:
-                facet_params = {
-                    'or': {
-                        'filters': [{'term': {'tags': tag}}
-                                    for tag in filter_tags],
-                        '_cache': True,
```

Comment:
For reference: http://www.elasticsearch.org/guide/en/elasticsearch/reference/current/query-dsl-term-filter.html#_caching_18